### PR TITLE
Update WindowsBuild.md

### PR DESCRIPTION
**Pull request by compnerd**
Extract curl build instructions into a step.  This makes it easier to identify where the instructions are.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please smoke test

File: docs/WindowsBuild.md [hunk 1]
```diff
@@ -248,7 +250,17 @@ cmake --build S:\b\libdispatch
...
 ```
 
-### 11. Build libxml2
+### 11. Build curl
+
+```cmd
+pushd "S:\curl"
+.\buildconf.bat
+cd winbuild
+name /f Makefile.vc mode=static VC=15 MACHINE=x64
```
Comment (gwynne) [inline docs/WindowsBuild.md]: I suspect you meant {{nmake}} here?

**Review by gwynne — CHANGES_REQUESTED** [review]

File: docs/WindowsBuild.md (hunk 1 above)
Comment (compnerd) [reply docs/WindowsBuild.md]: Typing is hard, yes, I did.  Thank you.

**Review by gwynne — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please smoke test and merge

**Comment by compnerd** [comment]
@swift-ci please smoke test and merge

**Pull request merged**